apps/aeutil/lib/serialization.ex

```diff
@@ -762,4 +771,30 @@ defmodule Aeutil.Serialization do
   def get_version(OracleQuery), do: {:ok, 1}
   def get_version(SignedTx), do: {:ok, 1}
   def get_version(ver), do: {:error, "#{__MODULE__} : Unknown Struct version: #{inspect(ver)}"}
+
```

Comment:
why is all of this commented out?

Reply:
This is just an old idea that i had at the beginning of my work, will remove it.